microcosm_pubsub/codecs.py

```diff
@@ -105,7 +103,10 @@ def configure_pubsub_message_codecs(graph):
     default_schema_cls = graph.config.pubsub_message_codecs.default
     strict = graph.config.pubsub_message_codecs.strict
 
-    message_codecs = defaultdict(lambda: PubSubMessageCodec(default_schema_cls(strict=strict)))
+    if default_schema_cls:
```

Comment:
It's still possible to have a default codec, but in practice this has been a bad idea since it meant that you might publish a message and only have its media type get sent (instead of any of its content). So now you have to opt-in for this behavior.